includes/syslog.php

```diff
@@ -53,9 +53,16 @@ function process_syslog($entry, $update)
     }
 
     $entry['host'] = preg_replace('/^::ffff:/', '', $entry['host']);
-    if ($new_host = Config::get("syslog_xlate.{$entry['host']}")) {
-        $entry['host'] = $new_host;
+
+    $axlate = Config::get('syslog_xlate');
+    if (true === is_array($axlate)) {
+        foreach ($axlate as $xname => $xlate) {
```

Comment:
why loop?   just do `$entry['host'] = $axlate[$entry['host']] ?? $entry['host'];`

Reply:
you could even inline the $xlate variable

Reply:
ok, letzt take your code - it's more efficient.